imageroot/actions/create-module/05setenvs

```diff
@@ -31,6 +38,7 @@ def allocate_rtp_ports_range(size):
     agent.assert_exp(int(seq) > 0)
     return (seq - size, seq - 1)
 
+
```

Comment:
Please, cleanup the patch: it's already massive, let's try to simplify the diff :)